main.go

```diff
@@ -61,6 +63,11 @@ func main() {
 		globalLogger.Fatal().Err(err).Msg("Failed creating applications")
 	}
 
+	if validateConfig == true {
+		globalLogger.Info().Bool("valid", true).Msg("Configuration is valid")
```

Comment:
Printing valid=true implies that a invalid config would print false. Instead it would return with exit 1 and another error message. We should probably rework this to return false and not fatal when in check mode.

Reply:
Also this should be added to the CI/E2E test

Reply:
Just start the app with the validate flag in CI and check if the exit code / output is correct